libraries/AP_Scripting/examples/DroneCAN_test.lua

```diff
@@ -0,0 +1,261 @@
+--[[
+    example of creating and sending DroneCAN messages
+--]]
+---@diagnostic disable: param-type-mismatch
+
+local MAGNETICFIELDSTRENGTHHIRES_ID = 1043
+local MAGNETICFIELDSTRENGTHHIRES_SIGNATURE = uint64_t(0x3053EBE3, 0xD750286F)
+
+local RAWAIRDATA_ID = 1027
+local RAWAIRDATA_SIGNATURE = uint64_t(0xC77DF38B, 0xA122F5DA)
+
+local PARAM_GETSET_ID = 11
+local PARAM_GETSET_SIGNATURE = uint64_t(0xA7B622F9, 0x39D1A4D5)
+
+local NODESTATUS_ID = 341
+local NODESTATUS_SIGNATURE = uint64_t(0x0F0868D0, 0xC1A7C6F1)
+
+local MAV_SEVERITY = {EMERGENCY=0, ALERT=1, CRITICAL=2, ERROR=3, WARNING=4, NOTICE=5, INFO=6, DEBUG=7}
+
+local TARGET_NODE = Parameter('SCR_USER1')
+
+-- a handle we will use for broadcasting
+local dc_handle = DroneCAN_Handle(0)
+
+-- a handle for fetching parameters
+local param_handle = DroneCAN_Handle(0)
+param_handle:data_type(PARAM_GETSET_ID)
+param_handle:signature(PARAM_GETSET_SIGNATURE)
+
+--[[
+    setup subscription to NodeStatus
+--]]
+local nodestatus_handle = DroneCAN_Handle(0)
+nodestatus_handle:data_type(NODESTATUS_ID)
+nodestatus_handle:signature(NODESTATUS_SIGNATURE)
+nodestatus_handle:subscribe()
+
+--[[
+    setup subscription to raw air data
+--]]
+local airspeed_handle = DroneCAN_Handle(0)
```

Comment:
Should this be `DroneCAN:handle(0)` so we can add other methods to the `DroneCAN` object in the future.

Reply:
One example might be to get a list of the "seen" node IDs. Or to reset the saved unique ID in the database for a given  node.

Reply:
DroneCAN:handle(0) would normally imply DroneCAN is a singleton, which it isn't (there is one per bus). Should we still do it that way? Not sure how to set that up without a singleton

Reply:
I guess we could have a bus object, so you would do a "bus" and then have a "handle" method on the bus. So you wuold endup with:
```
DroneCAN:bus(0):handle()
```